textworld/generator/game.py

```diff
@@ -266,23 +322,21 @@ class Game:
     """
 
     def __init__(self, world: World, grammar: Optional[Grammar] = None,
-                 quests: Optional[List[Quest]] = None,
+                 quests: List[Quest] = [],
```

Comment:
I'm always scared of things like this, since the default argument value is shared across calls.  In this case it's safe, but you might as well just write `= ()` and then not have to worry :)

Reply:
Wow, I've never thought of defaulting to `()`. Agreed, no risk of modifying that default value this way.